projects/step-frontend/src/lib/modules/timeseries/modules/_common/components/resolution-picker/resolution-picker.component.ts

```diff
@@ -23,14 +25,36 @@ export class ResolutionPickerComponent {
     { label: 'Weeks', ms: 7 * 24 * 3_600_000 },
     { label: 'Months', ms: 31 * 24 * 3_600_000 },
   ];
-  selectedUnit: ResolutionPickerOption = this.UNITS[0];
-  inputValue?: string;
+  _selectedUnit: ResolutionPickerOption = this.UNITS[0];
+  _inputValue?: string;
```

Comment:
We're applying underscore prefix for injected fields
```suggestion
  selectedUnit: ResolutionPickerOption = this.UNITS[0];
  inputValue?: string;
```